lib/src/main/kotlin/com/swmansion/starknet/data/types/Transaction.kt

```diff
@@ -382,7 +382,7 @@ data class DeclareTransactionV2(
     val contractDefinition: Cairo1ContractDefinition? = null,
 ) : DeclareTransaction(), DeprecatedTransaction {
     @Throws(ConvertingToPayloadFailedException::class)
-    internal fun toPayload(): DeclareTransactionV2Payload {
```

Comment:
why this function will be no longer internal, what's the rationale?

Reply:
The change was unintentional and has been removed in the newest push.